lib/stdlib/src/epp.erl

```diff
@@ -232,6 +237,39 @@ format_error({warning,Term}) ->
     io_lib:format("-warning(~tp).", [Term]);
 format_error(E) -> file:format_error(E).
 
+-spec scan_file(FileName, Options) ->
+        {'ok', [Form], Extra} | {error, OpenError} when
+      FileName :: file:name(),
+      Options :: [{'includes', IncludePath :: [DirectoryName :: file:name()]} |
+		  {'source_name', SourceName :: file:name()} |
+		  {'macros', PredefMacros :: macros()} |
+		  {'default_encoding', DefEncoding :: source_encoding()}],
+      Form :: erl_scan:tokens() | {'error', ErrorInfo} | {'eof', Loc},
```

Comment:
dialyzer is unhappy over `erl_scan:tokens/0` not being exported. Can you export it? I figure it's just an oversight since it's documented and the closely related `erl_scan:token/0` is exported.